src/components/Provider/ProviderHeader/CopyTraderButton/CopyTraderButton.js

```diff
@@ -127,6 +131,16 @@ const CopyTraderButton = ({ provider }) => {
     );
   };
 
+  const checkIfCanBeDisconnected = () => {
+    if (userData.isAdmin) {
+      return true;
+    }
+    if (provider.isAdmin) {
+      return false;
+    }
```

Comment:
I think it' easier to read if we simply do:
```
return userData.isAdmin || !provider.isAdmin
```

Reply:
@cwagner22  done. thanks!